### DISCUSSION TITLE
Search for Marker

### DISCUSSION THREAD

**Discussion by DennisKjaerChristensen** [GMOD/jbrowse#1631] [General]
Searching for eg. marker Excalibur_c23239_961 against

https://urgi.versailles.inra.fr/jbrowseiwgsc/gmod_jbrowse/?data=myData%2FIWGSC_RefSeq_v2.1&loc=Chr2D%3A605351801..605543800&tracks=DNA%2CINFINIUM90K_SUMMARY%2CBreedWheat-280K%2CLowConfidenceGenesv2.1&highlight=

does not find anything 

![image](https://user-images.githubusercontent.com/108664273/177131177-0e34966a-d08f-4ddc-a157-592c04801746.png)

but the marker is cleary seen here

![image](https://user-images.githubusercontent.com/108664273/177130937-a3a46bf7-b83c-494c-9bd1-292be99bd29d.png)

Regards
Dennis

**Comment by cmdcolin** [comment]
Hi there,
It looks like this track was not indexed with generate-names.pl by their site admins (see https://urgi.versailles.inra.fr/jbrowseiwgsc/gmod_jbrowse/myData/IWGSC_RefSeq_v2.1/names/meta.json, the tracks list does not include infinium90k)

You could maybe request the site admins to re-run generate-names with this track indexed!
